packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -273,24 +320,14 @@ export default Component.extend({
   cellRendererPrefix: 'cell-renderers/',
 
   /**
-   * @property {Number} rowHeight - height in px of a single item
+   * @property {Number} estimateHeight - estimated height in px of a single row
    */
-  rowHeight: 30,
+  estimateHeight: 32,
 
   /**
-   * @property {Array} rowDimensions - indicates the dimensions for each row of data
+   * @property {Number} bufferSize - size of the buffer before and after the collection
    */
-  rowDimensions: computed('tableData', 'expandedIdx', function() {
-    let rowDimension = formatItemDimension(get(this, 'rowHeight'));
-
-    //Create a set of row dimensions for each row of data
-    let rowDimensions = new Array(get(this, 'tableData.length'));
-    for (let i = 0; i < rowDimensions.length; i++) {
-      rowDimensions[i] = rowDimension;
-    }
-
-    return rowDimensions;
-  }),
+  bufferSize: 0,
```

Comment:
Should we make this 10 or 20?

Reply:
It was flickery when not all rows had the same height. I can play with it a little more

Reply:
changed to 10